### PR TITLE
IC-2016 Referral Send must exclude inactive RAR requirements

### PR DESCRIPTION
**Pull request by mahalrmoj**
**What does this pull request do?**
Community Api should exclude inactive requirements when deriving the RAR requirement for a CRN sentence during the process of initiating a NSI in Delius. Also, any post request made for an NSI that has duplicates in Delius should result in a 400. The NSIs must be cleaned up in Delius before they can be acted upon.

**What is the intent behind these changes?**
Fix issues arising in production

**Breaking Changes**
None.

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/delius/service/RequirementService.java
Comment (sldblog) [inline src/main/java/uk/gov/justice/digital/delius/service/RequirementService.java:119]:
```suggestion
    // that has a main category of F - rehab activity requirement. It is invalid for multiple active ones to exist.
```

**Review by sldblog — COMMENTED** [review]
LGTM, but will defer to code owners

File: src/main/java/uk/gov/justice/digital/delius/service/RequirementService.java
Comment (mahalrmoj) [reply src/main/java/uk/gov/justice/digital/delius/service/RequirementService.java]: Done

**Review by electrified — DISMISSED** [review]

**Review by electrified — APPROVED** [review]

**Pull request merged**
